tensorflow/python/training/experimental/mixed_precision.py

```diff
@@ -27,6 +31,44 @@
 from tensorflow.python.util.tf_export import tf_export
 
 
+
+@tf_export(v1=['train.experimental.auto_mixed_precision_scope'])
+@contextlib.contextmanager
+def auto_mixed_precision_scope(select=True):
```

Comment:
For consistency with `enable_mixed_precision_graph_rewrite`, maybe call this `mixed_precision_graph_rewrite_scope`?

Or, if you don't need to nest these scopes, maybe just have this disable mixed precision, without having the option to set it to True. Then it can be called `disable_mixed_precision_scope()`, or maybe `keep_as_float32`.

@azaks2 any suggestions for the name here? It will go under API review.

Reply:
Sounds good, I changed it to `mixed_precision_graph_rewrite_scope` for now.  If the API without a boolean argument is preferable, I can also add an `enable_mixed_precision_scope()` so we could keep the ability to nest.